plugins/module_utils/vcenter_utils.py

```diff
@@ -15,96 +15,116 @@
 import requests
 import atexit
 
-from pyVim import connect
+from pyvim import connect
 from pyVmomi import vmodl
 from pyVmomi import vim
 
-def establish_vcenter_connection(module, vCenter_host, username, password):
-    '''
+
+def establish_vcenter_connection(module, vCenter_host, username, password, ignore_ssl_verification):
+    """
```

Comment:
Bug Fixed - 2731783 - Getting "info from vCenter: [SSL: CERTIFICATE_VERIFY_FAILED]" when deploying 2nd and 3rd NSX-T managers

Logic - 
Added a new flag ignore_ssl_verification to decide whether SSL will be used to connect to VC or not.